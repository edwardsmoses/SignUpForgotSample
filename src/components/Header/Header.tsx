import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

import * as Colors from '../../styles/Color';

interface HeaderProps {
    title: string;
}

const Footer: React.FC<HeaderProps> = ({ title }) => {
    return (
        <View style={styles.containerStyle}>
            <Text style={styles.headerStyle}>{title}</Text>
        </View>
    );
};

export default Footer;

const styles = StyleSheet.create({
    containerStyle: {
        alignContent: 'center',
        justifyContent: 'center',
        padding: 20,
    },
    headerStyle: {
        color: Colors.Blue,
        fontSize: 25,
        fontWeight: '600',
        textAlign: 'center',
    },
});
