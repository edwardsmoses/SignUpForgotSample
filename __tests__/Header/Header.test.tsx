import { render, cleanup } from 'react-native-testing-library';
import React from 'react';
import Header from '../../src/components/Header/Header';

afterEach(cleanup);

describe('<Footer />', () => {
    it('should match snapshot', () => {
        const rendered = render(<Header title="Finish Signing Up" />).toJSON();

        expect(rendered).toMatchSnapshot();
    });
});
