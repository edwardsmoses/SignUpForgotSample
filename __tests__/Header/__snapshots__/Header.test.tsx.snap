// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match snapshot 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "justifyContent": "center",
      "padding": 20,
    }
  }
>
  <Text
    style={
      Object {
        "color": "#53a1e8",
        "fontSize": 25,
        "fontWeight": "600",
        "textAlign": "center",
      }
    }
  >
    Finish Signing Up
  </Text>
</View>
`;
